src/storage-mover/azext_storage_mover/_help.py

```diff
@@ -26,8 +38,19 @@
 examples:
     - name: endpoint create-for-nfs
       text: >
-        az storage-mover endpoint create-for-nfs -g {rg} --storage-mover-name {mover_name} -n {endpoint_nfs}
-        --description endpointDesc --export exportfolder --nfs-version NFSv4 --host {vm_ip}
+        az storage-mover endpoint create-for-nfs -g "rg" --storage-mover-name "mover_name" -n "endpoint_nfs"
+        --description endpointDesc --export exportfolder --nfs-version NFSv4 --host "vm_ip"
+"""
+
+helps['storage-mover endpoint create-for-smb'] = """
+type: command
+short-summary: Creates an Endpoint resource for smb mount.
+examples:
+    - name: endpoint create-for-smb
```

Comment:
When trying to create an SMB endpoint with credentials, it's created without the credentials:

**E.g.:**
`az storage-mover endpoint create-for-smb --endpoint-name smb-endpoint --storage-mover-name cli-sm --description hi --share-name smb --username-uri https://smb-demo-kv.vault.azure.net/secrets/username --password-uri https://smb-demo-kv.vault.azure.net/secrets/password --host 127.0.0.1 --resource-group afek-cli-tests`

`az storage-mover endpoint show --resource-group afek-cli-tests --storage-mover-name cli-sm --endpoint-name smb-endpoint`

**Response:**
`{ 
  "id": "/subscriptions/8e146654-89da-4a29-b7bc-ad1fa6175fec/resourceGroups/afek-cli-tests/providers/Microsoft.StorageMover/storageMovers/cli-sm/endpoints/smb-endpoint",
  "name": "smb-endpoint",
  "properties": {
    "description": "hi",
    "endpointType": "SmbMount",
    "host": "127.0.0.1",
    "provisioningState": "Succeeded",
    "shareName": "smb"
  },
  "resourceGroup": "afek-cli-tests",
  "systemData": {
    "createdAt": "2023-07-05T12:34:06.0365472Z",
    "createdBy": "[afekhazan@microsoft.com](mailto:afekhazan@microsoft.com)",
    "createdByType": "User",
    "lastModifiedAt": "2023-07-05T12:34:06.0365472Z",
    "lastModifiedBy": "[afekhazan@microsoft.com](mailto:afekhazan@microsoft.com)",
    "lastModifiedByType": "User"
  },
  "type": "microsoft.storagemover/storagemovers/endpoints"
}`

Reply:
Fixed.